orc-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(ORC_RT_HEADERS
+    orc-rt-c/orc-rt.h
+)
+
+add_library(orc-rt-headers INTERFACE)
+target_include_directories(orc-rt-headers INTERFACE
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+set_property(TARGET orc-rt-headers
+    PROPERTY PUBLIC_HEADER ${ORC_RT_HEADERS}
+)
+install(DIRECTORY ./
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/
+    FILES_MATCHING PATTERN "*.h"
```

Comment:
Should this get `COMPONENT OrcRT_Development`?

Reply:
I'm not sure. As you can probably tell I'm pulling this together with copy-pasta.

These headers are only useful for clients developing code against the ORC runtime libraries, and I'd like to group them as such for install purposes. What's the best-practice way to describe that grouping that's compatible with CMake 3.20? Is it just adding `COMPONENT OrcRT_Development` to the `install` arguments?

Reply:
I'd group things into logical components at least. Finer-grained can be better to at least help document what goes where, but is also more work. If managing more components is worth it, do so, otherwise a single "things needed for the SDK" component is usually sufficient.

Reply:
Thanks!